### ISSUE TITLE
README missing informations

### ISSUE THREAD

**Issue by pameruoso** [rspring/Esphome-Growatt#1]
D2 (GPIO4) on D1 mini to RE on module (flow control, orange in my setup)

I think you missed mentioning that RE and DE should be shorted, only looking closely to the picture where the MAX485 is downside I noticed you soldered the 2 pins... is that correct?

**Comment by rspring** [comment]
Indeed, (see: https://electronics.stackexchange.com/questions/153500/correctly-using-re-and-de-with-rs485). Strange thing is that my MAX485 convertor module died just a few weeks ago. I therefor replaced it with a new module. But on the new module I did not think about shortening DE and RE. Still the new module works fine without these pins shortened.
EDIT: It might very well be that I have used that module for transmit purpose in the past in another project. As for this project only data is received and not transmitted, I believe connecting DE with RE is irrelevant.

**Comment by pameruoso** [comment]
Same here… I was doing my testing and never shorted DE and RE… after doing my tests I was wiring everything for the “stable” setup and suddenly it stopped working…. “No response received” messages in the log …. But this could be me doing things wrong or really DE and RE with this max485 must be shorted to allow flow control to work… to be honest im thinking to switch to a rs-485 with no flow control pin needed…
